src/Akka.Streams.Kafka/Stages/Consumers/Abstract/KafkaSourceStage.cs

```diff
@@ -27,19 +28,25 @@ public abstract class KafkaSourceStage<K, V, TMessage> : GraphStageWithMateriali
         
         /// <inheritdoc />
         public override SourceShape<TMessage> Shape { get; }
+        
+        protected bool AutoCreateTopics { get; }
```

Comment:
Would this get set to true per this setup? https://github.com/akkadotnet/Akka.Streams.Kafka/issues/228

https://github.com/Aaronontheweb/Akka.Streams.Kafka.Reproduction/blob/75d88a432b747437dd1c4b20fa30e3d766727294/src/Akka.Streams.Kafka.Reproduction.Consumer/Program.cs#L33-L42

And 

https://github.com/Aaronontheweb/Akka.Streams.Kafka.Reproduction/blob/75d88a432b747437dd1c4b20fa30e3d766727294/src/Akka.Streams.Kafka.Reproduction.Consumer/Program.cs#L73

i.e. the `WithProperty` method setting this to `true`?

Reply:
No

Reply:
That might be an issue then. Need to detect that no matter how the user sets it.

Reply:
Is there a way we can check via the underlying Kafka client itself rather than our settings class?

Reply:
(What's the best way to do this?)

Reply:
Code moved. The code will try its best to look up this value from the ConsumerSettings, the only exception is when you use external source actor, since it would not have access to the original settings.

Reply:
We can document that edge case.